docs/reference/composer/api-modification.md

```diff
@@ -20,11 +20,14 @@ app.platformatic.addComposerOnRouteHook('/users/{id}', ['GET'], routeOptions =>
     }
   }
 
-  routeOptions.preSerialization = async (request, reply, payload) => {
-    return {
+  async function onComposerResponse (request, reply, body) {
+    const payload = await body.json()
+    const newPayload = {
       firstName: payload.first_name,
       lastName: payload.last_name
     }
+    reply.send(newPayload)
   }
+  routeOptions.config.onComposerResponse = onComposerResponse
```

Comment:
I think it would be helpful if the existence and purpose of `routeOptions.config.onComposerResponse` was also briefly explained in these docs (outside of this code example).

Reply:
done

Reply:
Great, this is really clear now 👍🏻